File: SkillMatrix.Domain/Skills/Models/Skill.cs
using SkillMatrix.Domain.Users.Models;
using System;
using System.Collections.Generic;

namespace SkillMatrix.Domain.Skills.Models
{
    public class Skill : BaseEntity
    {
        public long SkillId { get; set; }
        public string SkillName { get; set; }
        public long SkillCategoryId { get; set; }
        public virtual SkillCategory SkillCategory { get; set; }
        

    }
}

File: SkillMatrix/ClientApp/src/app/admin/skills/skill.ts
export interface Skill {
    id: number;
    skillName: string | null;
    skillCategoryName: string | null;
    skillCategoryId: number | null;
}

export interface AddSkill {
    id: number | undefined;
    skillName: string | null;
    skillCategoryId: number | null;
}
